src/components/ApplicationViews.js

```diff
@@ -1,74 +1,51 @@
 import React from "react"
 import { Route } from "react-router-dom"
-import { LocationProvider } from "./location/LocationProvider"
-import { AnimalProvider } from "./animal/AnimalProvider"
 import { LocationList } from "./location/LocationList"
 import { AnimalList } from "./animal/AnimalList"
-import { CustomerProvider } from "./customer/CustomerProvider.js"
 import { CustomerList } from "./customer/CustomerList.js"
-import { EmployeeProvider } from "./employee/EmployeeProvider";
 import { EmployeeList } from "./employee/EmployeeList";
 import { EmployeeForm } from "./employee/EmployeeForm.js"
 import { AnimalForm } from "./animal/AnimalForm.js"
 import { EmployeeDetail } from "./employee/EmployeeDetail.js"
 import { LocationDetail } from "./location/LocationDetail.js"
 import { AnimalDetails } from "./animal/AnimalDetail.js"
-import { AnimalSearch } from "./animal/AnimalSearch.js"
 
 export const ApplicationViews = () => {
     return (
         <>
-            <LocationProvider>
-                <EmployeeProvider>
-                    <AnimalProvider>
-                        <Route exact path="/">
-                            <LocationList />
-                        </Route>
+            <Route exact path="/">
+                <LocationList />
+            </Route>
 
-                        <Route path="/locations/:locationId(\d+)" render={
-                            props => <LocationDetail {...props} />
-                        } />
-                    </AnimalProvider>
-                </EmployeeProvider>
-            </LocationProvider>
-
-            <AnimalProvider>
-                <CustomerProvider>
-                    <LocationProvider>
-                        <Route exact path="/animals" render={(props) => {
-                            return <>
-                                <main className="animalContainer">
-                                    <h1>Animals</h1>
+            <Route path="/locations/:locationId(\d+)" render={
```

Comment:
You can remove all usage of `render={props =>...`

For example, this one can be rewritten as...

```
<Route path="/locations/:locationId(\d+)">
    <LocationDetail />
</Route>
```

This is because the new `useParams()` hook from React Router eliminates the need to pass props to the child component.